core/src/main/java/io/questdb/griffin/engine/functions/GroupByFunction.java

```diff
@@ -74,6 +74,28 @@ default boolean earlyExit(MapValue mapValue) {
 
     int getValueIndex();
 
+    /**
+     * Called for group by function cloned to be used in different threads of parallel execution.
```

Comment:
This would need an overall design description of our parallel GROUP BY. I agree that it would be great to have it, but not sure if it should be put here.